### PR TITLE
feat(plugin-chart-pivot-table): multiple improvements

### PR DESCRIPTION
**Pull request by kgabryje**
List of changes:
1. Header and totals row are sticky
2. Added aggregation function name to "Total" label
3. Changed labels from "Totals" to "Total"
4. Moved "Show total" checkboxes to Data tab
5. "Show total" checkboxes are unchecked by default

https://user-images.githubusercontent.com/15073128/126642403-c2f4bc96-77e9-49e4-8a2c-fa033df06ff6.mov

CC: @junlincc @rusackas

### CODE REVIEW

File: plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx [hunk 1]
```diff
@@ -39,9 +40,9 @@ import {
...
 
 const Styles = styled.div<PivotTableStylesProps>`
-  padding: ${({ theme }) => theme.gridUnit * 4}px;
-  height: ${({ height }) => height}px;
-  width: ${({ width }) => width}px;
+  margin: ${({ margin }) => margin}px;
+  height: ${({ height, margin }) => height - margin * 2}px;
+  width: ${({ width, margin }) => width - margin * 2}px;
   overflow-y: scroll;
   }
```
Comment (michael-s-molina) [inline plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx:47]:
```suggestion
  ${({ height, width, margin }) => `
      margin: ${margin}px;
      height: ${height - margin * 2}px;
      width: ${width - margin * 2}px;
      overflow-y: scroll;
 `}
```

**Review by zhaoyongjie — APPROVED** [review]
the codes LGTM! thanks, Kamil.

File: plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx (hunk 1 above)
Comment (kgabryje) [reply plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx]: Done, thanks!

**Comment by jaimefr** [comment]
@kgabryje thanks for the great contribution. One question, why does sticky was only applied to the "top row header" and not to the "left column header"?

**Pull request merged**